config/locales/views/graders/en.yml

```diff
@@ -17,9 +17,10 @@ en:
     group_submission_no_files: One or more of the groups selected has no files submitted. Grader not assigned.
     help: Assign graders to specific student groups and criteria. By default, graders will be assigned to all criteria.
     hide_unassigned_criteria: Only show assigned criteria to graders (if unselected, all graders will be able to see all criteria)
-    no_grader_assigned: 'Default grader: course instructor(s)'
+    no_grader_assigned: "Default grader: course instructor(s)"
     select_a_criterion: Select at least one criterion
     select_a_grader: Select at least one grader to assign.
-    skip_empty_submissions: Do not assign graders to groups with empty submissions (requires that submissions are collected)
+    skip_empty_submissions: Assign graders to groups with empty submissions
+    skip_empty_submissions_tooltip: Requires that submissions are collected
```

Comment:
Reword to "If unchecked, groups with no files submitted are skipped if submissions have already been collected. This option has no effect if submissions are not yet collected."